src/Symfony/Bundle/FrameworkBundle/Routing/RouteLoaderInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing;
+
+/**
+ * Marker interface for service route loaders.
+ */
+interface RouteLoaderInterface
```

Comment:
IMHO, this interface should be in the FWB because it would have 0 usage in the component + its particular name would interfere with real route loaders there.

Reply:
However, I still think that prefixing this interface and the temporary container with "Service" would be better to be consistent with the "type: service".